adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1172,37 +1172,33 @@ class ADAMContext private (@transient val sc: SparkContext) extends Serializable
   /**
    * Loads variant annotations stored in VCF format.
    *
-   * @see loadVcf
-   * @see loadVariantAnnotations
-   *
    * @param filePath The path to the VCF file(s) to load annotations from.
-   * @return Returns DatabaseVariantAnnotationRDD.
+   * @return Returns VariantAnnotationRDD.
    */
-  def loadVcfAnnotations(filePath: String): DatabaseVariantAnnotationRDD = {
-    loadVcf(filePath).toDatabaseVariantAnnotationRDD
+  def loadVcfAnnotations(
+    filePath: String): VariantAnnotationRDD = {
+    loadVcf(filePath).toVariantAnnotationRDD
```

Comment:
Running SnpEff on the VCF files we are already using for unit tests ends up being not too interesting, with all intragenic variants.  It might take a little thinking to generate a more useful VCF file, say with variants right at intron/exon boundaries of a gene with a lot of splice variants, for example.